tests/src/Interop/COM/NETClients/Primitives/ArrayTests.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace ComClient
+{
+    using CoreFXTestLibrary;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    class ArrayTests
+    {
+        static private readonly IEnumerable<int> BaseData = Enumerable.Range(0, 10);
+
+        private readonly Server.Contract.IArrayTesting server;
+        private readonly double expectedMean;
+
+        public ArrayTests()
+        {
+            this.server = (Server.Contract.IArrayTesting)new Server.Contract.Servers.ArrayTestingClass();
```

Comment:
Nit: Please use Object instead of interface.. it will cover a little more scenario(compiler need to add cast) if using object and feel nature for C# user.